evaluator/evaluate.go

```diff
@@ -178,7 +178,7 @@ func evaluateInDatabase(ctx context.Context, event *mqueue.PlatformEvent, invent
 		return nil, nil, nil
 	}
 
-	vmaasData, err := evaluateWithVmaas(tx, updatesData, system, event.GetAccountName())
+	vmaasData, err := evaluateWithVmaas(tx, updatesData, system, event.GetAccountName(), event.GetOrgID())
```

Comment:
When you pass `event.GetAccountName(), event.GetOrgID()` to the function would not be easier to pass whole `&event` instead and call `event.GetAccountName(), event.GetOrgID()` inside?
I mean it makes argument clean and passing pointer is also less data.